### PR TITLE
Add coding phase article

### PR DESCRIPTION
**Pull request by jorgepiloto**
This pull request adds a new post related to the end of the coding phase as suggested by the Open Astronomy organization :relaxed:

### CODE REVIEW

File: content/articles/2019-05-27-Bonding-period-ends.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Title: Coding phase starts!
+Date: 2019-05-27 11:30
+Category: GSCOC
```
Comment (astrojuanlu) [inline content/articles/2019-05-27-Bonding-period-ends.md]:
```suggestion
Category: GSOC
```

File: content/articles/2019-05-27-Bonding-period-ends.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-05-27-poliastro-gsoc-jorge
```
Comment (astrojuanlu) [inline content/articles/2019-05-27-Bonding-period-ends.md]: I recommend:

```suggestion
slug: 2019-05-27-coding-phase-starts
```

And also renaming the file

File: content/articles/2019-05-27-Bonding-period-ends.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+lang: en
+Author: Jorge Martínez
```
Comment (astrojuanlu) [inline content/articles/2019-05-27-Bonding-period-ends.md]:
```suggestion
Author: Jorge Martínez Garrido
```

To make it the same to https://github.com/poliastro/poliastro.github.io/blame/sources/content/articles/2019-05-10-poliastro-gsoc-jorge.md

File: content/articles/2019-05-27-Bonding-period-ends.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+corporum coelestium in sectionibus conicis Solem ambientium" a way to transform
+Kepler's algorithm into a third order polynomial. This led to an amazing fast
+convergence when solving near parabolic orbits.
+
+On Battin 1999 and improved Gauss algorithm is explained. However after several
+hours of coding and debugging I have not been able to find why it does not
+converge to the expected solution. I will come back to it after some time and
+for sure make a post about this algorithm, because it is amazing how Gauss
+was able to find a solution just by using convergent series!
+
```
Comment (astrojuanlu) [inline content/articles/2019-05-27-Bonding-period-ends.md]: Remove extra whitespace

File: content/articles/2019-05-27-Bonding-period-ends.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+mentors, other developers and their future ideas on the software...
+
+As I said in my last post, I started contributing to poliastro several months
+ago, even before GSOC was announced. Therefore, and since I am taking my final
+exams, all my efforts were towards passing them.
+
+But today the coding phase starts! That means we all should start working on
+our proposal features.
+
+## The coding phase begins!
```
Comment (astrojuanlu) [inline content/articles/2019-05-27-Bonding-period-ends.md]: I think we can get rid of this section! More or less everybody knows, and it's very generic :)

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Made some comments!

File: content/articles/2019-05-27-Bonding-period-ends.md (hunk 2 above)
Comment (jorgepiloto) [reply content/articles/2019-05-27-Bonding-period-ends.md]: Oh sory! I thought this section called `slug` was something related to the RSS :sweat:

**Review by astrojuanlu — APPROVED** [review]

**Pull request merged**